src/app.ts

```diff
@@ -80,3 +80,5 @@ fs.writeFileSync(
 logger.info(
   `Exported ${outPackage.profiles.length} profile(s) and ${outPackage.extensions.length} extension(s).`
 );
+
+logger.info(`There was a total of ${stats.numError} error(s) and ${stats.numWarn} warning(s).`);
```

Comment:
Can we reformat how this gets displayed?  I'm thinking something a little cleaner and easier to visually parse might be nice.  For example, we could replace the two logging statements above so that they produce something more like this:

```
Profiles:    12
Extensions:  5
Errors:      0
Warnings:    0
```

Or something fancier:
```
+============================+
|          RESULTS           |
|----------------------------|
| Profiles:   12             |
| Extensions:  5             |
+============================+
    Errors: 0  Warnings: 0
```

Reply:
The fancier version seems a bit much, to me (I can change it if you want though), but I will update to report results more cleanly!